test/framework/gardenerframework.go

```diff
@@ -100,6 +100,7 @@ func (f *GardenerFramework) BeforeEach() {
 func validateGardenerConfig(cfg *GardenerConfig) {
 	if cfg == nil {
 		ginkgo.Fail("no gardener framework configuration provided")
+		return // make linters happy
```

Comment:
Hmm, just a question - we have many usages of `ginkgo.Fail` without a return. What is the the issue with it?

Reply:
Don't know TBH, it just complained in those two funcs, when I ran `make check`.